src/main/resources/peer-review-transform.xsl

```diff
@@ -153,8 +153,17 @@
         <xsl:apply-templates select="supplementary-material" />
       </dl>
     </xsl:if>
+
+    <xsl:variable name="review-doi-url">
+      <xsl:text>https://doi.org/10.1371/journal.</xsl:text><xsl:value-of select="../@id"/>
+    </xsl:variable>
+    <div class="review__doi">
```

Comment:
Why two underscores?

Reply:
Following the convention Sam established: https://github.com/PLOS/wombat/blob/develop/src/main/resources/peer-review-transform.xsl

Reply:
Excellent, ty!